src/helpers/__tests__/ci-env/appveyor.json

```diff
@@ -6,7 +6,7 @@
       "APPVEYOR_BUILD_ID": "appveyor-build-id",
       "APPVEYOR_BUILD_NUMBER": "appveyor-pipeline-number",
       "APPVEYOR_REPO_BRANCH": "master",
-      "APPVEYOR_REPO_COMMIT": "appveyor-repo-commit",
+      "APPVEYOR_REPO_COMMIT": "b9f0fb3fdbb94c9d24b2c75b49663122a529e123",
```

Comment:
Are we actually doing anything with hashes to make sure they're valid? Otherwise, isn't it just string comparison?

Reply:
It's just string comparison. Eric changed them to actually look like hashes in the files that we use to generate these fixtures, but it's mostly an aesthetic change at this point.